### DISCUSSION TITLE
React 18 is currently in beta stage. Are there any plans on it?

### DISCUSSION THREAD

**Discussion by nosferatu500** [ant-design/ant-design#32885] [General]
The current version of ant.design (4.17) doesn't work with React 18. When I try to interact with the dropdown menu, it just freezing my app.

**Comment by luanlucho** [comment]
Antd can get so much benefit from server components. Please consider  prioritizing it in your timeline.

**Reply by nosferatu500** [reply]
Sorry but I don't understand what are you talking about...
